sass/modules/_forms.scss

```diff
@@ -14,6 +14,9 @@
  * permissions and limitations under the License.
  */
 
+/* This is the url for our static content service */
+$staticImage: '/static-content/image/file';
```

Comment:
default this to ../img for backwards compatibility.  We'll need to update this var in the places that we want the new location.

Also we'll want to update the images themselves to use the /resource/ endpoint with the image identifier.